hanan_shirin_qays/moivesapp/src/Components/Main.js

```diff
@@ -0,0 +1,17 @@
+import React, { useState, useEffect } from "react";
+import MoviesGrid from "./MoviesGrid";
+
+function Main(props) {
```

Comment:
extract the props like 
```suggestion
function Main({ query, movies }) {
```